### DISCUSSION TITLE
Having multiple Postal servers as opposed to one

### DISCUSSION THREAD

**Discussion by impirecom** [postalserver/postal#2554] [Help with using Postal]
Greetings marvelous Postal Community,

I have setup 3 postal servers each with different names and domains. Each have its own set of IPs.

The reason I set up 3 servers as opposed to 1 is to protect the domain reputation. This is just in case one server get black listed, at least I have 2 other servers to work with. However, managing 1 server would be easier than 3.

Does running on one server using the combined IPs is not a good choice? Does black list watch dogs mainly flag the sending server names, as well as the IPs?

Thank you in advance for your help.

**Comment by willpower232** [comment]
You have essentially manually implemented IP Pools https://docs.postalserver.io/features/ip-pools although others have been struggling with them according to recent issues and discussions so at least you know your setup works :sweat_smile: 

I can imagine blocklists working from both IP addresses and domain names given the wide variety of companies who will let you run servers on their IP addresses.

**Reply by impirecom** [reply]
Thanks @willpower232. We own the IP blocks and that made it easier.

For sender reputation, do you know if Blacklist watch dogs care about mail server names or are they primarily focusing on the IP addresses only?

**Reply by willpower232** [reply]
I have only ever been in situations where the IP address needed removing from a list so I imagine that is more common than the name but naturally organisations tend to be a bit secretive about how their blocking mechanisms work so who can say for sure.
